extensions/amp-subscriptions/0.1/amp-subscriptions.js

```diff
@@ -166,6 +186,16 @@ export class SubscriptionService {
     this.initialize_().then(() => {
       this.renderer_.toggleLoading(true);
 
+      user().assert(this.pageConfig_, 'Page config is null');
+
+      user().assert(this.serviceConfig_['services'],
```

Comment:
Keep in mind, in `object['services']` is not the same as `typedef.services` in general case. You either have to make it an `Object` or actually remap.